pkg/client/fake/client_test.go

```diff
@@ -2695,6 +2746,81 @@ var _ = Describe("Fake client", func() {
 	}
 })
 
+type WithSchemalessSpecList struct {
+	*metav1.ListMeta
+	*metav1.TypeMeta
+	Items []*WithPointerMeta
+}
+
+func (t *WithSchemalessSpecList) DeepCopy() *WithSchemalessSpecList {
+	l := &WithSchemalessSpecList{
+		ListMeta: t.ListMeta.DeepCopy(),
+	}
+	if t.TypeMeta != nil {
+		l.TypeMeta = &metav1.TypeMeta{
+
+			APIVersion: t.APIVersion,
+
+			Kind: t.Kind,
+		}
+	}
+	for _, item := range t.Items {
+		l.Items = append(l.Items, item.DeepCopy())
+	}
+
+	return l
+}
+
+func (t *WithSchemalessSpecList) DeepCopyObject() runtime.Object {
+	return t.DeepCopy()
+}
+
+type Schemaless map[string]interface{}
+
+type WithSchemalessSpec struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec Schemaless `json:"spec,omitempty"`
+}
+
+func (t *WithSchemalessSpec) DeepCopy() *WithSchemalessSpec {
+	w := &WithSchemalessSpec{
+		ObjectMeta: *t.ObjectMeta.DeepCopy(),
+	}
+	w.TypeMeta = metav1.TypeMeta{
+		APIVersion: t.APIVersion,
+		Kind:       t.Kind,
+	}
+	t.Spec.DeepCopyInto(&w.Spec)
+
+	return w
+}
+
+func (t *WithSchemalessSpec) DeepCopyObject() runtime.Object {
+	return t.DeepCopy()
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *Schemaless) DeepCopyInto(out *Schemaless) {
+	if *in != nil {
+		*out = make(Schemaless, len(*in))
+		for key := range *in {
+			(*out)[key] = (*in)[key]
+		}
+	}
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new DynamicMap.
```

Comment:
looks like I forgot to rename from my own struct 😅 
```suggestion
// DeepCopy copies the receiver, creating a new Schemaless.
```